core/render/html/utils.py

```diff
@@ -31,6 +36,16 @@ def jinjaGlobalFilter(f):
     return f
 
 
+def jinjaGlobalTest(f):
+    """
+    Decorator, marks a function as a Jinja2 test.
+
+    The method name can start with "test_" to avoid name conflicts.
+    """
+    __jinjaTests_[f.__name__.lstrip("test_")] = f
```

Comment:
I'm a little unhappy with this. What do you think of providing a decorator `jinjaGlobalTest` that supports an alternative operand for the name, like so:
```python
@jinjaGlobalTest  # registers admin as admin
def admin(render, value):
	return value is not None and "admin" in value["access"]

@jinjaGlobalTest("dict")  # registers test_dict as dict
def test_dict(render: Render, value: Any) -> bool:
    """Jinja2 test: Return True if the object is a dict."""
    return isinstance(value, dict)
```

An alternative suggestions would be that tests must start with `test_`, which should raise an assertion error when not the case.

What do you think?

Reply:
I like this idea :+1:. Please check out my latest commit.